### PR TITLE
Fgts 203 polish all levels

### PR DESCRIPTION
**Pull request by fheysel**
Adds health bugs to game
Adds more birds to first level
Adds rats to second level
Fixes sawblade bug which causes it to collide with other sawblades, pushing them around the map

### CODE REVIEW

File: godotproject/Level0.tscn [hunk 1]
```diff
@@ -57,6 +58,7 @@ __meta__ = {
 motion_mirroring = Vector2( 320, 0 )
 
 [node name="Sprite" type="Sprite" parent="ParallaxBackground/ParallaxLayerSky"]
+position = Vector2( -4, 0 )
```
Comment (charasyn) [inline godotproject/Level0.tscn]: Was this changed intentionally? If not can you revert

File: godotproject/Tiles/Level2/Computer.png.import [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+[params]
+
+compress/mode=0
+compress/lossy_quality=0.7
+compress/hdr_mode=0
+compress/bptc_ldr=0
+compress/normal_map=0
+flags/repeat=0
+flags/filter=true
```
Comment (charasyn) [inline godotproject/Tiles/Level2/Computer.png.import]:
```suggestion
flags/filter=false
```

**Review by charasyn — CHANGES_REQUESTED** [review]
A few really minor changes

**Comment by fheysel** [comment]
Approved by @charasyn

**Pull request merged**